packages/paste-website/src/pages/components/form-pill-group/index.mdx

```diff
@@ -0,0 +1,434 @@
+- If dismissable, a Pill can be removed by pressing the delete or backspace key
+
+<Box marginBottom="space70">
+  <AspectRatio ratio="16:9">
+    <iframe
+      src="https://www.loom.com/embed/1bea986ec6a24cd494950e2c12f10562"
+      frameborder="0"
+      webkitallowfullscreen
+      mozallowfullscreen
+      allowfullscreen
+      title="Silent demo on keyboard navigation"
+    ></iframe>
+  </AspectRatio>
+</Box>
+
+#### Display Pill vs Form Pill
+
+There are some very important semantic differences between a Display Pill and Form Pill group. Below is a table explaining the main differences to get a better understanding of when to use each type of Pill.
+
+<FormPillVsDisplayPillTable />
+
+## Examples
+
+### Basic Form Pill Group
+
+The basic Form Pill Group can be composed by adding `FormPill` components as children of `FormPillGroup`. These must be used with the `useFormPillState` hook, and the returned state should be spread onto each component.
+
+This will provide you with some internal state management and keyboard navigation.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {defaultExample}
+</LivePreview>
+
+### Selectable Form Pill Group
+
+Outside of keyboard navigation, Form Pills are presentational. If you would like to make the Pill Group provide selection options, you can manage the state of those pills and seemlessly merge it with the state return from the `useFormPillState` hook.
+
+Track which pill is selected in a separate store of state. When rendering the Pill Group, cross reference the pills with the selected state, and conditoinally set `selected` on each `FormPill` that requires it.
+
+The `onSelect` event handler will fire when ever the pill is clicked, or the spacebar or enter key is pressed. Use this to respond to your users selection interactions.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {selectableExample}
+</LivePreview>
+
+### Removable Form Pill Group
+
+For removable Form Pills, because the Form Pill Group is largely presentational, you can provide removable functionality by managing the state of the rendered Pills. By responding to user interactions, hooking into the `onDismiss` event handler, you are able to update the rendered state of the Form Pill Group and selectively remove pills from the collection.
+
+The `onDismiss` event handler will fire when a user clicks on the close x, or presses their backspace or delete key when foused on a pill.
+
+<LivePreview scope={{FormPill, FormPillGroup, useFormPillState, CalendarIcon}} noInline>
+  {removableExample}
+</LivePreview>
+
+## Composition Notes
+
+### When to use a Form Pill Group
+
+Use Form Pill Groups when you are editing a collection of data within a form. It can be used to represent selection across multiple fields such as filtering, or from a single field like a combobox.
```

Comment:
We don't actually have an example of them being used in a form on these docs. I feel like having one would legitimize this ask.

Reply:
Oh my... 🤦🏼

Reply:
Actually I kind of take that back. They are in form elements, and a form can be a form of one field. So these are nice isolated examples of just the component on its own to make it super explicit that it is independent of everything else.

I would like maybe one more example though potentially as a composite. Maybe filtering?

Reply:
Yeah, having an example of some sort (which I think filtering is a good one) would help because at the moment, the examples on both this and display pill group look really similar. The more apparent we can make the differentiation btw these two components, the better.

Reply:
yesss +1 to an example, even if it's in the do/don't section only

If it's paired with the Multiselect Combobox primitive, it'd be nice to have a copy-paste-able example for people.

Reply:
Updated the primitive combobox example to use pills and imported that here